### PR TITLE
add port method

### PR DESCRIPTION
**Pull request by taozuhong**

### CODE REVIEW

File: include/sybdb.h [hunk 1]
```diff
@@ -1285,6 +1285,8 @@ RETCODE dbsetlversion (LOGINREC * login, BYTE version);
 #define DBSETDBNAME		14
 #define DBSETLDBNAME(x,y)	dbsetlname((x), (y), DBSETDBNAME)
 #define DBSETLVERSION(login, version) dbsetlversion((login), (version))
+#define DBSETPORT		15
```
Comment (freddy77) [inline include/sybdb.h:1288]: If it does not come from Sybase/Microsoft should be put as an extension below.

File: src/dblib/dblib.c [hunk 2]
```diff
@@ -854,7 +854,6 @@ dbsetllong(LOGINREC * login, long value, int which)
 	}
 }
 
-#if defined(DBLIB_UNIMPLEMENTED)
```
Comment (freddy77) [inline src/dblib/dblib.c:857]: Maybe also change the comment
```
 * \retval FAIL anything other than \c DBSETHIER was passed for \a which.  
```
to something more generic like
```
 * \retval FAIL if invalid field in \a which or invalid \a value.  
```

File: src/dblib/dblib.c [hunk 3]
```diff
@@ -877,14 +876,17 @@ dbsetlshort(LOGINREC * login, int value, int which)
 	}
 
 	switch (which) {
-	case DBSETHIER:
+	case DBSETPORT:
+		tds_set_port(login->tds_login, value);
+		return SUCCEED;
+		break;
+	// case DBSETHIER:
```
Comment (freddy77) [inline src/dblib/dblib.c:883]: Do not use C++ style comments

**Comment by taozuhong** [comment]
Merge all review suggestions into on commit (pls see #529)

**Pull request closed without merging**